src/modules/ChatListModule/ChatListModule.tsx

```diff
@@ -1,34 +1,27 @@
 import ChatComponent from "../../components/ChatComponent/ChatComponent";
 import SearchBar from "../../shared/UI/SearchBar/SearchBar";
 import AddChatComponent from "../../components/AddChatComponent/AddChatComponent";
-import { useEffect, useState } from "react";
+import { useEffect } from "react";
 import "./ChatListModule.scss";
-import ShortChatType from "../../shared/types/ShortChatType";
-import client from "../../shared/api/BaseAxiosInstance";
 import { List, ListItem } from "@mui/material";
+import { useChatState } from "../../shared/states/ChatStates";
+import ShortChatType from "../../shared/types/ShortChatType";
 
 const ChatListModule = () => {
-  const [selectedIndex, SetSelectedIndex] = useState(-1);
-  const [chatList, SetChatList] = useState([] as ShortChatType[]);
-  const [filteredChatList, SetFilteredChatList] = useState(
-    [] as ShortChatType[]
-  );
+  const [selectedChat, SetSelectedChat] = useChatState((state) => [
```

Comment:
const  {currentChat, setSelectedChat,  filteredChatList, setFilteredChatList, chats, fetchChats} = useChatState(state => ({
currentChat: state.currentChat,
updateCurrentChat: state.updateCurrentChat
........
})

Reply:
Try to keep original names of the variables and methods across the app